### PR TITLE
Transition testnet docs to mainnet docs

### PR DESCRIPTION
**Pull request by vitsalis**

### CODE REVIEW

File: docs/user-guides/phase1-mainnet.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+- [To participate covenant committee signing](https://github.com/babylonlabs-io/covenant-signer/blob/v0.2.x/README.md).
+
+## For LST Projects and Custodies <a id="lst-custodies"></a>
+
+- [Specification of all the transactions](https://github.com/babylonlabs-io/babylon/blob/dev/v0.9.x/transaction-impl-spec.md),
+which is for those who want to construct Bitcoin staking related transactions by
+themselves instead of using our code or dapps.
+
+- [Swagger API doc](https://staking-api.babylonlabs.io/swagger/index.html#),
```
Comment (gitferry) [inline docs/user-guides/phase1-mainnet.md:46]: This is not reachable. Should we specify the network?

File: docs/user-guides/phase1-mainnet.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+- [To become a finality provider](https://github.com/babylonlabs-io/networks/tree/main/bbn-1/finality-providers).
+
+## For Covenant Committee Signers <a id="covenant-signer"></a>
+
+- [To participate covenant committee signing](https://github.com/babylonlabs-io/covenant-signer/blob/v0.2.x/README.md).
+
+## For LST Projects and Custodies <a id="lst-custodies"></a>
+
+- [Specification of all the transactions](https://github.com/babylonlabs-io/babylon/blob/dev/v0.9.x/transaction-impl-spec.md),
```
Comment (gitferry) [inline docs/user-guides/phase1-mainnet.md:42]:
```suggestion
- [Specification of all the transactions](https://github.com/babylonlabs-io/babylon/blob/v0.9.x/docs/transaction-impl-spec.md),
```

File: docs/user-guides/phase1-mainnet.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+## For Wallets <a id="wallets"></a>
+
+- [Our staking web dapp](https://github.com/babylonlabs-io/simple-staking).
+
+- [Wallet integration guide](https://github.com/babylonlabs-io/networks/blob/main/bbn-1/integration/wallet.md).
+
+- [Swagger API doc](https://staking-api.babylonlabs.io/swagger/index.html#),
+which is for those who want to build their own BTC staking frontend but want to use our backend.
+
+- [Staking backend service build guide](https://github.com/babylonlabs-io/networks/blob/main/bbn-1/integration/staking-backend.md),
```
Comment (gitferry) [inline docs/user-guides/phase1-mainnet.md:26]: can use `v0.2.x` after the integration pr is merged

File: docs/user-guides/phase1-mainnet.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+💻 Official instruction list for participanting in Bitcoin Staking Phase-1
+Mainnet.
+
+---
+
+## For Wallets <a id="wallets"></a>
+
+- [Our staking web dapp](https://github.com/babylonlabs-io/simple-staking).
+
+- [Wallet integration guide](https://github.com/babylonlabs-io/networks/blob/main/bbn-1/integration/wallet.md).
```
Comment (gitferry) [inline docs/user-guides/phase1-mainnet.md:21]: can use v0.2.x after the integration pr is merged

File: docs/user-guides/phase1-mainnet.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+- [Swagger API doc](https://staking-api.babylonlabs.io/swagger/index.html#),
```
Comment (gitferry) [inline docs/user-guides/phase1-mainnet.md:23]: This is not reachable. Should we specify the network?

**Review by gitferry — APPROVED** [review]
Some comments about links

File: docs/user-guides/phase1-mainnet.md (hunk 4 above)
Comment (vitsalis) [reply docs/user-guides/phase1-mainnet.md:21]: Would like to keep the main as v0.2.x is only related to the params.

File: docs/user-guides/phase1-mainnet.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+which is for those who want to construct Bitcoin staking related transactions by
+themselves instead of using our code or dapps.
+Note that such transactions should be using the correct [global
+parameters](https://github.com/babylonlabs-io/networks/blob/main/bbn-1/parameters).
+
+- [Swagger API doc](https://staking-api.babylonlabs.io/swagger/index.html#),
+which is for those who want to build their own BTC staking frontend but want to
+use our backend.
+
+- Our staking web dapp as a [reference implementation](https://github.com/babylonlabs-io/simple-staking).
```
[the remark this answers is not in the document]
Comment (vitsalis) [reply docs/user-guides/phase1-mainnet.md]: done

**Pull request merged**